HW_6/HW_5a.py

```diff
@@ -0,0 +1,144 @@
+
+
+class PrivatAdv(Article):
+    def __init__(self, header, text, expiration_date):
+        Article.__init__(self, header=header, text=text)
+        self.expiration_date = expiration_date
+
+    def format_text(self, header, text, expiration_date):
+        return f"{header}----------" \
+               f"\n{text}\n" \
+               f"Actual until: {expiration_date}, {(expiration_date - datetime.date.today()).days} days left." \
+               f"\n-----------------------\n\n"
+
+
+class Consider(Article):
+    def __init__(self, header, text, score):
+        Article.__init__(self, header=header, text=text)
+        self.score = score
+
+    def format_text(self, header, text, score):
+        return f"{header}-------------- " \
+               f"\n{text}\n" \
+               f"Probability of truth:{score}%" \
+               f"\n------------------------------\n\n"
+
+
+def write_article():
+    input_versions = ('News', 'PrivatAdv', 'Consider')
+    header = input('Enter the topic you are interested in:\nNews\nPrivatAdv\nConsider\n:')
+
+    while header not in input_versions:
+        print('Please select correct topic:')
+        header = input('News\nPrivatAdv\nConsider\n:')
+
+    print("Type your text here. Press Enter twice to exit from text adding mode")
+    input_text = ''
+    while True:
+        new_line = input()
+        if new_line != '':
+            input_text += new_line + '\n'
+        else:
+            input_text = input_text[:-1]
+            break
+    text = normalization.normalization(input_text)
+    if header == 'News':
+        if len(text) == 0:
+            user_input = input("Enter the path of your file: ")
+            if os.path.exists(user_input):
+                input_file = open(user_input, 'r+')
+            else:
+                print("I did not find the file at, " + str(user_input) + " I'll use mine")
+                input_file = open('default_news.txt', "r")
+                parsed_text = input_file.read().split('\n\n')
+                for i in parsed_text:
+                    text = i[0:]
+                    location = 'Default location'
+                    story = News(header, text, location)
+                    story.print_to_file(story.format_text(header, text, location), 'sum_news.txt')
+            input_file.close()
+            os.remove('default_news.txt')
+        else:
+            location = input('Enter location \n:')
+            story = News(header, text, location)
+            story.print_to_file(story.format_text(header, text, location), 'sum_news.txt')
+    elif header == 'PrivatAdv':
+        if len(text) == 0:
+            user_input = input("Enter the path of your file: ")
+            if os.path.exists(user_input):
+                input_file = open(user_input, 'r+')
+            else:
+                print("I did not find the file at, " + str(user_input) + "I'll use mine")
+                input_file = open('default_advs.txt', "r")
```

Comment:
Also, a good practice is the creation of config files, which contain all names/paths of files, not related to code.